taskboot/build.py

```diff
@@ -48,18 +66,22 @@ def build_image(target, args):
             config.docker['password'],
         )
 
-    logger.info('Will produce image {}'.format(tag))
+        tags = gen_docker_images(base_image, args.tag, registry)
+    else:
+        tags = gen_docker_images(base_image, args.tag, args.registry)
```

Comment:
@La0 Do you think we should use `args.registry` in all cases instead of using either the config registry when pushing or the CLI flag value when building only?

Reply:
Problem is that the docker auth (in config) is tied to the registry too...
But we need to be consistent, so you can use `args.registry` everywhere and discard the config one.